### PR TITLE
Fix/overwrite testing

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

File: src/variables.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@ $rise-blue: #0081BA;
 $start-color: #1f71a2;
 $stop-color: #045888;
 $danger: #d9534f;
+$google-red: #df4a32;
```
Comment (RV-Adrian) [inline src/variables.scss]: Placed into Variables .scss, out of Buttons .scss

File: src/ui-components/file-selector.scss [hunk 2]
```diff
@@ -213,8 +213,8 @@
   font-size: 28px;
 }
 
-#storageSelectorModal, .storage-app  {
-  .btn-group {
-    display: inline;
-  }
-}
+// #storageSelectorModal, .storage-app  {
```
Comment (RV-Adrian) [inline src/ui-components/file-selector.scss]: I could not find anywhere .storage-app class is used

File: src/bootstrap-overwrites/_panels.scss [hunk 3]
```diff
@@ -99,6 +99,9 @@
   }
 }
 
+
+// Core Cleanup - Card Created
```
Comment (RV-Adrian) [inline src/bootstrap-overwrites/_panels.scss]: Pending card action

File: src/bootstrap-overwrites/_navbar.scss [hunk 4]
```diff
@@ -1,5 +1,4 @@
 .navbar {
-	position: relative;
```
Comment (RV-Adrian) [inline src/bootstrap-overwrites/_navbar.scss]: Redundant, from Bootstrap

File: src/bootstrap-overwrites/_modals.scss [hunk 5]
```diff
@@ -77,7 +77,7 @@
   }
 }
 
-@media (min-width: 768px) {
+@media (min-width: $screen-sm-min) {
```
Comment (RV-Adrian) [inline src/bootstrap-overwrites/_modals.scss]: Starting to use only variables for responsive

File: src/bootstrap-overwrites/_mixins.scss [hunk 6]
```diff
@@ -49,6 +54,14 @@
   }
 }
 
+@mixin button-size($padding-vertical, $padding-horizontal, $font-size, $line-height, $border-radius, $font-weight) {
```
Comment (RV-Adrian) [inline src/bootstrap-overwrites/_mixins.scss]: We don't use this Mixin, even though it's established in Bootstrap. Dropped it here for clarity.

File: src/bootstrap-overwrites/_code.scss [hunk 7]
```diff
@@ -10,5 +10,5 @@
 		// background-color: @pre-bg;
 		// border: 1px solid @pre-border-color;
 		// border-radius: @border-radius-base;
-		overflow: auto;
+		// overflow: auto;
```
Comment (RV-Adrian) [inline src/bootstrap-overwrites/_code.scss]: Could not see this used anywhere?

File: src/bootstrap-overwrites/_buttons.scss [hunk 8]
```diff
@@ -1,48 +1,42 @@
...
+    outline: 0;
+    -webkit-transition: all 0.2s;
+    transition: all 0.2s;
+    -webkit-font-smoothing: subpixel-antialiased;
 }
 
 .btn-lg {
-  font-size: 16px;
-  font-weight: 400;
+    @include button-size(10px, 16px, 16px, 1.33, 6px, 400);
```
Comment (RV-Adrian) [inline src/bootstrap-overwrites/_buttons.scss]: Starting to use 'size' Mixin here

File: src/bootstrap-overwrites/_alerts.scss [hunk 9]
```diff
@@ -1,16 +1,18 @@
...
-  @include alert-variant(#F8FFD6, #E9F19C, $text-color, $text-color);
-  display: block;
-  h2 { margin-top: 0; }
-  &:hover,
-  &:active {
-    color: $text-color;
-  }
-}
\ No newline at end of file
+// .alert-warning {
```
Comment (RV-Adrian) [inline src/bootstrap-overwrites/_alerts.scss]: After testing, using the base 'alert' mixins from Bootstrap produce the same results.

**Comment by RV-Adrian** [comment]
@ghislaineguerin 
Please review the changes here as part of https://trello.com/c/v9XXgZZK/1896-ux-css-cleanup-overwrites-2

I've tested these changes against Apps,Store,Storage and Common Header. I didn't see any bugs as a result, but review my inline comments and let me know if you're aware of any impact? Thanks!

**Comment by ghislaineguerin** [comment]
👍 looks good

File: src/variables.scss (hunk 1 above)
Comment (ghislaineguerin) [inline src/variables.scss]: @RV-Adrian awesome, thanks

File: src/ui-components/file-selector.scss (hunk 2 above)
Comment (ghislaineguerin) [inline src/ui-components/file-selector.scss]: @RV-Adrian I think it's no longer used.

**Comment by ghislaineguerin** [comment]
👍

**Pull request merged**